holoviews/operation/datashader.py

```diff
@@ -176,6 +194,15 @@ def _process(self, element, key=None):
         category = agg_fn.column if isinstance(agg_fn, ds.count_cat) else None
         x, y, data, glyph = self.get_agg_data(element, category)
 
+        if x is None or y is None:
+            x0, x1 = self.p.x_range or (-0.5, 0.5)
+            y0, y1 = self.p.y_range or (-0.5, 0.5)
+            xc = np.linspace(x0, x1, self.p.width)
+            yc = np.linspace(y0, y1, self.p.height)
+            xarray = xr.DataArray(np.full((self.p.height, self.p.width), np.NaN, dtype=np.float32),
+                                  dims=['y', 'x'], coords={'x': xc, 'y': yc})
```

Comment:
Not unless you want to add a is_all_nan switches to ``Image`` Elements. It's two distinct operations.